content/guides/caching.md

```diff
@@ -4,284 +4,358 @@ sort: 11
 
-Notice that **vendor chunk has the same filename**, and **so does the manifest** since we’ve extracted the manifest chunk!
+__src/index.js__
 
+``` diff
+  import _ from 'lodash';
+- import Print from './print';
++ // import Print from './print';
 
-## Manifest inlining
+  function component() {
+    var element = document.createElement('div');
 
-Inlining the chunk manifest and webpack runtime (to prevent extra HTTP requests), depends on your server setup. There is a nice [walkthrough for Rails-based projects](https://brigade.engineering/setting-up-webpack-with-rails-c62aea149679). For server-side rendering in Node.js you can use [webpack-isomorphic-tools](https://github.com/halt-hammerzeit/webpack-isomorphic-tools).
+    // Lodash, currently included via a script, is required for this line to work
+    // Lodash, now imported by this script
+    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+-   element.onClick = Print.bind(null, 'Hello webpack!');
++   // element.onClick = Print.bind(null, 'Hello webpack!');
+
+    return element;
+  }
+
+  document.body.appendChild(component());
+```
+
+And finally run our build again:
+
+``` bash
+Hash: 37e1358f135c0b992f72
+Version: webpack 3.3.0
+Time: 1557ms
+                          Asset       Size  Chunks                    Chunk Names
+ vendor.eed6dcc3b30cfa138aaa.js     541 kB       0  [emitted]  [big]  vendor
+   main.fc7f38e648da79db2aba.js  891 bytes       1  [emitted]         main
+runtime.bb5820632fb66c3fb357.js    5.85 kB       2  [emitted]         runtime
+                     index.html  352 bytes          [emitted]
+[3IRH] (webpack)/buildin/module.js 517 bytes {0} [built]
+[DuR2] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [0] multi lodash 28 bytes {0} [built]
+[lVK7] ./src/index.js 427 bytes {1} [built]
+    + 1 hidden module
+```
 
-T> If your application doesn’t rely on any server-side rendering, it’s often enough to generate a single `index.html` file for your application. To do so, use i.e. [`HtmlWebpackPlugin`](https://github.com/ampedandwired/html-webpack-plugin) in combination with [`ScriptExtHtmlWebpackPlugin`](https://github.com/numical/script-ext-html-webpack-plugin) or [`InlineManifestWebpackPlugin`](https://github.com/szrenwei/inline-manifest-webpack-plugin). It will simplify the setup dramatically.
+We can see that both builds yielded `eed6dcc3b30cfa138aaa` in the `vendor` bundle's filename.
 
 
-## References
+## Conclusion
 
-* https://medium.com/@okonetchnikov/long-term-caching-of-static-assets-with-webpack-1ecb139adb95#.vtwnssps4
-* https://gist.github.com/sokra/ff1b0290282bfa2c037bdb6dcca1a7aa
-* https://github.com/webpack/webpack/issues/1315
-* https://github.com/webpack/webpack.js.org/issues/652
-* https://presentations.survivejs.com/advanced-webpack/
+Caching gets messy. Plain and simple. However the walk-through above should give you a running start to deploying consistent, cache-able assets. See the _Further Reading_ section below to learn more.
```

Comment:
Yeah, I guess it's ok to append `able` to words even if you it's not an actual variation of that word. Using the `-` prevents spell checkers from throwing an error but I guess it does read better without it.